src/app/app.component.ts

```diff
@@ -17,21 +17,17 @@ export class AppComponent {
     private analyticsService: AnalyticsService,
     private router: Router) {
     this.viewContainerRef = viewContainerRef;
-    // analytics handled through gatg() automatically manages sessions, visits, clicks, etc
-    gtag('config', 'G-EEHE8GEBH4');
-    gtag('event', 'measur_app_open', {
-      'measur_platform': 'measur-web'
-    });
-    
-    this.router.events.subscribe(event => {
-      if (event instanceof NavigationEnd) {
-        let path: string = environment.production? event.urlAfterRedirects : 'testing-web';
-        path = this.analyticsService.getPageWithoutId(path);
-        gtag('event', 'page_view', {
-          'page_path': path,
-          'measur_platform': 'measur-web',
-        });
-      }
-    });
+    if (environment.production) {
+      // analytics handled through gatg() automatically manages sessions, visits, clicks, etc
+      gtag('config', 'G-EEHE8GEBH4');
+      this.analyticsService.sendEvent('measur_app_open', undefined);
```

Comment:
1. I'm in favor of leaving all gtag calls on this page for grouping sake.
2. Do you know for sure that there is nothing lost by not sending the measur_app_open via the gtag? Again trying to recall going through the documentation and the confusing specifics about session setup, start, and those stats.